src/features/Apiexplorer/RequestResponseRenderer/index.tsx

```diff
@@ -7,7 +7,7 @@ import useAuthContext from '@site/src/hooks/useAuthContext';
 import { useCallback } from 'react';
 import BrowserOnly from '@docusaurus/BrowserOnly';
 import { Circles } from 'react-loader-spinner';
-interface IResponseRendererProps<T extends TSocketEndpointNames> {
+export interface IResponseRendererProps<T extends TSocketEndpointNames> {
```

Comment:
Do we need to export this? I do not see it being used anywhere.

Reply:
Yes we will use this for tests